racket/collects/ffi/unsafe/define.rkt

```diff
@@ -8,21 +8,19 @@
          provide-protected
          make-not-available)
 
-(define (make-not-available id)
-  (lambda ()
-    (lambda args
-      (error id "implementation not found; ~a"
-             (if (null? args)
-                 "no arguments provided"
-                 (apply
-                  string-append
-                  "arguments:"
-                  (let loop ([args args])
-                    (if (null? args)
-                        null
-                        (cons (format " ~e"
-                                      (car args))
-                              (loop (cdr args)))))))))))
+(define (((make-not-available id)) . args)
```

Comment:
This might be less clear in the new version -- maybe 0-argument functions should be a special case?

Reply:
I think that makes sense. Opened https://github.com/jackfirth/resyntax/issues/61 for this.